litestar/app.py

```diff
@@ -971,18 +947,6 @@ def get_membership_details(group_id: int, user_id: int) -> None:
 
         return join_paths(output)
 
-    @property
-    def route_handler_method_view(self) -> dict[str, list[str]]:
```

Comment:
This was only used to construct the routes and isn't *really* necessary, as you can achieve the same with the `route_handler_method_map` attribute. 

Need to add a deprecation warning for this in 2.x